db/migrate/20200423154018_change_teen_program_to_transition_aged_youth.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeTeenProgramToTransitionAgedYouth < ActiveRecord::Migration[6.0]
+  def change
+    rename_column :casa_cases, :teen_program_eligible, :transition_aged_youth
```

Comment:
lgtm, we don't have uptime requirements right now so we don't need two-step deploys :)
We did confirm that migrations need manual intervention for now- @tundal45 is working on it in https://github.com/rubyforgood/casa/issues/142